src/emu/debugger/src/imgui_debugger.cpp

```diff
@@ -1096,53 +1096,60 @@ namespace eka2l1 {
             // Align to center!
             ImGui::SameLine((ImGui::GetWindowSize().x - (BUTTON_SIZE.x * 2)) / 2);
 
-            if (ImGuiButtonToggle("Yes", BUTTON_SIZE, device_wizard_state.should_continue)) {
-                device_wizard_state.stage = static_cast<device_wizard::device_wizard_stage>(static_cast<int>(device_wizard_state.stage) + 1);
-                device_wizard_state.should_continue = false;
+            if (device_wizard_state.stage == device_wizard::ENDING) {
+                if (ImGui::Button("OK", BUTTON_SIZE)) {
+                    should_show_install_device_wizard = false;
+                    device_wizard_state.stage = device_wizard::WELCOME_MESSAGE;
```

Comment:
A little bit confusing, but change this to FINAL_FOR_REAL, since at that state all things are reinitialized for next installation session (see [here](https://github.com/EKA2L1/EKA2L1/blob/2b689354c5101566edb6efcdd7e559d5e5d90c33/src/emu/debugger/src/imgui_debugger.cpp#L995))

Reply:
changes pushed